legend-engine-language-pure-dsl-service-execution/src/test/java/org/finos/legend/engine/language/pure/dsl/service/execution/TestServiceRunner.java

```diff
@@ -376,6 +376,38 @@ public void SimpleM2MServiceRunnerForRecursiveExecution()
         Assert.assertEquals("{\"wheels\":55}", result);
     }
 
+    private static class SimpleM2MServiceRunnerForDateTimeSerialization extends AbstractServicePlanExecutor
+    {
+        SimpleM2MServiceRunnerForDateTimeSerialization(String function)
+        {
+            super("test::Service", buildPlanForFetchFunction("/org/finos/legend/engine/pure/dsl/service/execution/test/simpleM2MService.pure", function), true);
+        }
+
+        @Override
+        public List<ServiceVariable> getServiceVariables()
+        {
+            return Collections.emptyList();
+        }
+    }
+
+    @Test
+    public void SimpleM2MServiceRunnerForDateTimeSerialization()
```

Comment:
can we please update the test to work with few diff formats and compare it without format. Maybe including timezone

Reply:
Added tests with few other dateTime formats and with no format